Bloxstrap/Models/IPInfoResponse.cs

```diff
@@ -0,0 +1,35 @@
+namespace Bloxstrap.Models
+{
+    public class IPInfoResponse
+    {
+        [JsonPropertyName("city")]
+        public string City { get; set; }
+
+        [JsonPropertyName("country")]
+        public string Country { get; set; }
+
+        [JsonPropertyName("region")]
+        public string Region { get; set; }
+
+        [JsonPropertyName("ip")]
+        public string IP { get; set; }

```

Comment:
Only include properties in the response class that are used by the original code.